### DISCUSSION TITLE
Build is successfull but application error

### DISCUSSION THREAD

**Discussion by pdaccache** [apache/cordova#446] [Q&A]
Dears, my sdk target version was 30 and it was working fine i had to upgrade to api level 33, so i have installed android@12.0.1 and the build is successfull and when emulation i am not able to start the app with application error http://localhost:8080/index.html error, i have tried everything and i am not able to make it work.
It was working normally i haven't changed anything beside the api level in config.xml please your help.

**Comment by breautek** [comment]
Please include relevant details such as:

- The full error message
- any stacktraces
- Relevant code and/or configurations
